pkg/util/conditions/getter.go

```diff
@@ -0,0 +1,260 @@
+/*
+Copyright 2020 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conditions
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	clusterv1 "github.com/kcp-dev/kcp/pkg/apis/conditions/v1alpha1"
+)
+
+// Getter interface defines methods that a Cluster API object should implement in order to
```

Comment:
Global comment: replace any code comment references to Cluster API appropriately